docs/architecture/policy_credentialtype_scope.md

```diff
@@ -0,0 +1,153 @@
+# Mapping Credentials
+
+## Definition of terms
+
+- Credential type: a [W3C Verifiable Credential type](https://w3c.github.io/vc-data-model/#types), for
+  example `DismantlerCredential`. For every credential type there must be a resolvable context, which contains further
+  schema and type information. Credentials can have multiple types. This is comparable to a class definition in OOP
+  languages. Also referred to as: CredentialType, type
+
+- Credential: an instance of a credential type, i.e. one materialization of a particular schema. This is comparable to a
+  class instance in OOP languages. Also referred to as: vc, VC, VerifiableCredential
+
+- Framework credential: a credential having the credential type `UseCaseFrameworkCondition`. All Framework credentials
+  are the same type and therefore adhere to the same schema. Also referred to as: Framework agreement credential
+
+- Use case type: refers to the `credentialSubject.type` property of the `UseCaseFrameworkCondition` type. The use case
+  type discriminates which framework agreement use case the credential attests to. It is **not** a separate
+  VerifiableCredential type! For
+  example: [PCF Use case credential](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/3.%20Verifiable%20Credentials/CX-Credentials/Standardized%20CX-Credential.md#pcf-use-case-credential).
+  Specifically, `subject.type = PcfCredential`. Also referred to as: use case
+
+- Contract version: refers to the `credentialSubject.contractVersion` property and is expected to be
+  a [SemVer string](https://semver.org/). The `contractVersion` discriminates which framework agreement use case version
+  the credential attests to. Also referred to as a `framework agreement version`.
+
+- Credential version: a credential version does not exist as a first-class concept.
+
+## Referencing credentials in policies
+
+Please check the
+syntax [here](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/edc/policy.definitions.md)
```

Comment:
I know I get punished now to raise the topic here - but sorry had to document it somewhere :D I suggest to do an update in the policy syntax for "DismantlerConstraint"

We should change from 
`"leftOperand": "Dismantler.activityType",`
to
`"leftOperand": "CompanyRole.Dismantler",`

I fully understand if we want to discuss this in another round - this should for sure not block this pr

Reply:
as you said. that's food for another discussion/issue/pr, because _this_ PR is about documenting the mapping, not the policy expression DSL syntax.